src/pages/Calculator.tsx

```diff
@@ -66,6 +66,7 @@ export const Calculator = (): React.ReactElement => {
   }
 
   const resetForm = () => {
+    window.scrollTo(0, 0)
```

Comment:
My two cents is that it would be better to scroll to the top of the calculator section rather than the top of the page. But if you wanna keep it this way, let's just merge it.